src/uiScreens_bagl.c

```diff
@@ -653,11 +653,23 @@ void ui_displayNetworkParamsScreen(
 	STATIC_ASSERT(!IS_SIGNED(networkId), "signed type for %u");
 	STATIC_ASSERT(sizeof(protocolMagic) <= sizeof(unsigned), "oversized type for %u");
 	STATIC_ASSERT(!IS_SIGNED(protocolMagic), "signed type for %u");
+
+	#ifdef APP_FEATURE_BYRON_PROTOCOL_MAGIC_CHECK
 	snprintf(
 	        networkParams, SIZEOF(networkParams),
 	        "network id %u / protocol magic %u",
 	        networkId, protocolMagic
 	);
+	#else
+	// if the protocol magic check is not enabled,
+	// displaying the protocol magic might be misleading,
+	// so we must not show it
+	snprintf(
+	        networkParams, SIZEOF(networkParams),
+	        "network id %u",
+	        networkId
+	);
```

Comment:
Nit: The comment talks about displaying protocol magic, when network id is being displayed.

Reply:
Ah just my confusion, ignore this.